new-lamassu-admin/src/pages/OperatorInfo/ReceiptPrinting/ReceiptPrinting.styles.js

```diff
@@ -1,10 +1,12 @@
+import { booleanPropertiesTableStyles } from 'src/components/booleanPropertiesTable/BooleanPropertiesTable.styles'
+
+const { rowWrapper } = booleanPropertiesTableStyles
```

Comment:
Lets not import styles from another component, better to just rewrite the style here for now.

Reply:
Okay!